CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+## How to contribute
+
+If you are interested in joining the project, then please review our [Projects Board](https://github.com/hackforla/VRMS/projects/2) to see if anything interests you. If so, then please do the following.
+
+1. Review the Hack for LA Getting Started Guide https://www.hackforla.org/getting-started.
+
+1. Join the VRMS weekly meetings https://www.vrms.io/.
+
+1. Reach out to our project leads to discuss how you would like to contribute.
+
+### Committing changes overview
+
+1. Find a ticket in the [Prioritized Backlog](https://github.com/hackforla/VRMS/projects/2) and assign yourself to the ticket.
+
+1. Fork the repo.
+
+1. Make a branch off of `development`.
+
+1. Commit and push your changes to your forked repository.
+
+1. From the `VRMS` repository, create a Pull Request from your branch into the VRMS `developnent` branch.
+
+1. Any changes will need approval and merging from a project leads.
+
+1. Bring an update of your changes to the weekly VRMS team checkin.
+
+### Forking and cloning the repository with proper security
+
+#### Step 1: Become a member of the repository Team
```

Comment:
It intersects with my first comment about joining the project. So it is covered by https://github.com/hackforla/VRMS/wiki/Join-the-Team 
I mean that contributing to the project and joining the team is the same.

Reply:
👍